components/fxa-client/ffi/src/lib.rs

```diff
@@ -286,6 +286,17 @@ pub extern "C" fn fxa_get_session_token(handle: u64, error: &mut ExternError) ->
     ACCOUNTS.call_with_result_mut(error, handle, |fxa| fxa.get_session_token())
 }
 
+/// Check if OAuth refresh token is authorized
+/// Should be called before retrieving the access token
```

Comment:
I don't think this comment is necessarily true; it implies that you should call this any time you want to get an access token, but I think you'd only want to call it after you tried *and failed* to get an access token, in order to find out what might have gone wrong.